src/Sulu/Bundle/MarkupBundle/Listener/MarkupListener.php

```diff
@@ -39,12 +39,14 @@ public function __construct(array $markupParser)
      */
     public function replaceMarkup(FilterResponseEvent $event)
     {
-        $format = $event->getRequest()->getRequestFormat();
+        $format = $event->getRequest()->getRequestFormat(null);
```

Comment:
html is default but if no format is detected - also no markup-parser should be called.